packages/web-sdk/src/transports/otlp/payload/transform/transform.ts

```diff
@@ -94,7 +93,6 @@ function toLogRecord(transportItem: LogTransportItem): LogRecord {
       return toMeasurementLogRecord(transportItem as TransportItem<MeasurementEvent>);
     default:
       const error = `Unknown TransportItemType: ${type}`;
-      internalLogger.error(error);
```

Comment:
Hi @kpelelis 
I used the internalLogger in wrong way in this module.

**Problem was:**
I instantiated the logger myself in it's own file.
But here I can't access the Faro config which may include a custom logger defined by the user.

So I removed it and started investigating if I:
* Can receive the already configured logger form some kind of context 
* Or if I have to pass it down from the transport because each transport has the configured logger attached by default.

**Solution**
We have no context where I can get the logger from so I will pass it further down the tree starting from the transport.

Reply:
I am unresolving the thread just to keep track of the changes